templates/appliance/about.html

```diff
@@ -2,117 +2,151 @@
 
 {% block title %}About Ubuntu Appliances{% endblock %}
```

Comment:
Title should be "What is an Ubuntu Appliance?"

Reply:
done